File: src/test/resources/java/MathOperations.java
public class MathOperations {
    public int calculateResult1(int x, int y) {
        int sum = x + y;
        int product = x * y;
        int difference = x - y;
        return (sum + product - difference) + 10;
    }

    public int calculateResult2(int p, int q) {
        int difference = p - q;
        int product = p * q;
        int sum = p + q;
        return 2 * (sum + product - difference);
    }
}

File: src/test/resources/js/MathOperations.js
class MathOperations {
    calculateResult1(x, y) {
        const sum = x + y;
        const product = x * y;
        const difference = x - y;
        return (sum + product - difference) + 10;
    }

    calculateResult2(p, q) {
        const difference = p - q;
        const product = p * q;
        const sum = p + q;
        return 2 * (sum + product - difference);
    }
}